config/logger.config

```diff
@@ -0,0 +1,28 @@
+---
+version: 1
+
+disable_existing_loggers: False
+
+formatters:
+    simple:
+        format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+handlers:
+    console:
+        class: logging.StreamHandler
+        level: DEBUG
+        formatter: simple
+        stream: ext://sys.stdout
+
+    file_handler:
+        class: logging.handlers.RotatingFileHandler
+        level: WARNING  # By default set to WARNING, but can be changed on runtime
+        formatter: simple
+        filename: /var/log/arteria/siswrap.log
```

Comment:
Me and Steinar were talking about not putting the full path `/var/log/...` into the template config, as it will not work without first creating directories etc. which makes it difficult to just try things out. I suggest switching to `filename: siswrap.log` to handle this.